package.json

```diff
@@ -70,7 +71,8 @@
     "build:pp": "gatsby build --prefix-paths",
     "build:gh": "npm run clean && npm run build:pp && gh-pages -d public",
     "clean": "rm -rf public && rm -rf .cache",
-    "lint:js": "eslint --ext .js,.jsx .",
+    "lint": "npm run lint:js && npm run lint:md",
```

Comment:
je to spravne kontrolovat `.md` soubory taky? co si myslete?

Reply:
Kontrolu MD souborů bych do CI nedával a přemýšlím, zda i neodstranit nástroje (závislost a script) pro tuto kontrolu.